custom_components/solaredge_modbus_multi/number.py

```diff
@@ -208,25 +208,25 @@ def __init__(self, inverter, config_entry, coordinator):
 
     @property
     def unique_id(self) -> str:
-        return f"{self._platform.uid_base}_storedge_remote_command_timeout"
+        return f"{self._platform.uid_base}_storage_remote_command_timeout"
 
     @property
     def name(self) -> str:
-        return "Storedge Remote Command Timeout"
+        return "Command Timeout"
```

Comment:
Maybe "Storage Command Timeout" to demonstrate this applies to the "Storage Command" entity? All of the "select" entities are prepended with "storage". We should be consistent and prepend the numbers as well?

Reply:
Probably fine, the reason I shortened them as much as I did was for display on the device page.